.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 before_script:
   - npm install -g npm
   - npm cache clear
-  - npm install -g broccoli-cli
```

Comment:
```
npm run build
```
is using the system wide 
```
gulp
```
executable.
Seems that gulp is preinstalled on Travis, so this works.
Maybe we should try in travis.yml
```
npm install -g gulp
```
to be sure to have the newest versions?